### PR TITLE
sgml-common: fix html-doc install path and download url.

### PR DESCRIPTION
**Pull request by klausz65**

### CODE REVIEW

File: components/docbook/sgml-common/Makefile [hunk 1]
```diff
@@ -39,12 +38,8 @@ COMPONENT_PREP_ACTION +=(cd $(@D) && \
 # Missing files in build dir for configure without this.
 COMPONENT_PRE_CONFIGURE_ACTION =        ($(CLONEY) $(SOURCE_DIR) $(@D))
```
Comment (AndWac) [inline components/docbook/sgml-common/Makefile:41]: It looks like --disable-dependency-tracking is needed.

Comment (klausz65) [reply components/docbook/sgml-common/Makefile:41]: Hmm, not on oibuild:
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating bin/Makefile
config.status: creating bin/install-catalog
config.status: creating config/Makefile
config.status: creating doc/Makefile
config.status: creating doc/HTML/Makefile
config.status: creating doc/man/Makefile
config.status: creating doc/refentry/Makefile
config.status: creating sgml-iso-entities/Makefile
config.status: creating xml-iso-entities/Makefile
/usr/bin/touch /ws/klausz/oi-userland/components/docbook/sgml-common/build/amd64/.configured

Comment (AndWac) [reply components/docbook/sgml-common/Makefile:41]: But on oi.buildzone with Jenkins.

Comment (klausz65) [reply components/docbook/sgml-common/Makefile:41]: Okay added wanted option to Makefile

**Comment by AndWac** [comment]
Something seems to be broken. When I run
╰─➤  gmake REQUIRED_PACKAGES
cd /export/home/andreas/oi-userland/components/docbook/sgml-common/build/prototype/i386//usr; mv doc share
mv: cannot access doc
gmake: *** [Makefile:43: install] Error 2

**Pull request merged**